store/store_action.go

```diff
@@ -56,6 +56,7 @@ type CurrentSnap struct {
 	Block            []snap.Revision
 	Epoch            snap.Epoch
 	CohortKey        string
+	ValidationSets   [][]string
```

Comment:
If I understand the intent correctly, these are not just simple arrays of strings but structures with a well-defined semantic (snap-id, revision, tracking-channel and instance-key, as I read from the PR description). It would be nice if we could express this with a real structure or, if you think that's not worth it, I highly encourage you to add a comment before this field to explain the meaning of the various array positions.

Reply:
The description from the PR is about the other fields of the struct; the new ValidationSets field is about validation set keys (either primary keys or sequence keys) which are string arrays everywhere in our code.

I agree comment is a good idea, will add, thanks.

Reply:
ah-ah, then I indeed misunderstood :-) But thanks for adding the comment!